### DISCUSSION TITLE
Cannot Run official Winforms project

### DISCUSSION THREAD

**Discussion by bigboybamo** [dotnet/winforms#12079] [Q&A] [answered]
I forked and cloned the project, I was able to build the project but when open Visual Studio using `start-vs.cmd?`, I get the error 
<img width="316" alt="image" src="https://github.com/user-attachments/assets/5322970b-e2f2-486a-96b1-06b359ff30ab">
And then when I try to reload any project, I get the error. 
<img width="605" alt="image" src="https://github.com/user-attachments/assets/1c49ea7f-c895-4d07-8c99-060836da128e">

I'm not sure what else I need to do to run the project, looking for suggestions. Thanks.

**Comment by elachlan** [comment, marked as answer]
Visual studio preview version is needed as well I believe.

**Reply by bigboybamo** [reply]
Thanks, this resolved it for me
